docs/releases/deployment-process.rst

```diff
@@ -0,0 +1,60 @@
+Deployment Process
+==================
+
+Deployment and provisioning are currently managed in a separate repo.
```

Comment:
It is currently closed-source, but there is a JIRA ticket to consider whether to make it open source. I think that it would be really useful.